src/clj_json_patch/util.clj

```diff
@@ -48,34 +48,6 @@
         (get-patch-value val segs)
         val))))
 
-(defn set-patch-value
```

Comment:
this function was shadowed by another copy of it below, so i've just removed it.